src/core/components/select/Select.stories.tsx

```diff
@@ -0,0 +1,125 @@
+			mapping: {
+				undefined: undefined,
+				error: 'Please select your home state. This service is unavailable outside of the US.',
+			},
+			control: { type: 'radio' },
+		},
+		success: {
+			options: ['undefined', 'success'],
+			mapping: {
+				undefined: undefined,
+				success: 'This service is available in your state',
+			},
+			control: { type: 'radio' },
+		},
+	},
+	args: {
+		label: 'State',
+		optional: false,
+		hideLabel: false,
+		supporting: '',
+		error: '',
+		success: '',
+	},
+};
+
+const Template: Story<SelectProps> = (args: SelectProps) => (
+	<Select {...args}>
+		<Option value="">Select a state</Option>
+		<Option value="al">Alabama</Option>
+		<Option value="ca">California</Option>
+	</Select>
+);
+
+// *****************************************************************************
+
+export const Playground = Template.bind({});
+Playground.args = {
+	supporting: 'Leave blank if you are not within the US',
+};
+asPlayground(Playground);
+
+// *****************************************************************************
+
+export const DefaultLight = Template.bind({});
+asChromaticStory(DefaultLight);
+
+// *****************************************************************************
+
+export const ErrorWithMessageLight = Template.bind({});
+ErrorWithMessageLight.args = {
+	error: 'Please select your home state. This service is unavailable outside of the US.',
+};
+asChromaticStory(ErrorWithMessageLight);
+
+// *****************************************************************************
+
+export const VisuallyHideLabelLight = Template.bind({});
+VisuallyHideLabelLight.args = {
+	hideLabel: true,
+};
+asChromaticStory(VisuallyHideLabelLight);
+
+// *****************************************************************************
+
+export const OptionalLight = Template.bind({});
+OptionalLight.args = {
+	optional: true,
+};
+asChromaticStory(OptionalLight);
+
+// *****************************************************************************
+
+export const SuccessWithMessageLight = Template.bind({});
+SuccessWithMessageLight.args = {
+	success: 'This service is available in your state',
+};
+asChromaticStory(SuccessWithMessageLight);
+
+// *****************************************************************************
+
+export const SupportingTextLight = Template.bind({});
+SupportingTextLight.args = {
+	supporting: 'Leave blank if you are not within the US',
+};
+asChromaticStory(SupportingTextLight);
+
+// *****************************************************************************
+
+const message = css`
+	${textSans.medium()}
+`;
+
+export const ControlledExample = (args: SelectProps) => {
+	const [state, setState] = React.useState('al');
+
+	return (
+		<>
+			<Template
+				{...args}
+				value={state}
+				onChange={(event) => setState(event.target.value)}
+			/>
+			<span css={message}>
+				{state ? `You have selected: ${state}` : ''}
+			</span>
+		</>
+	);
+};
+asChromaticStory(ControlledExample);
```

Comment:
We can probably get rid of this story, it's not really doing anything useful on Chromatic. If we need to demonstrate a code example of a controlled component, we can direct people to the [story in source-shakedown](https://github.com/guardian/source-shakedown/blob/main/stories/select/controlled.tsx)